tests/testthat/_snaps/derive_vars_dtm.md

```diff
@@ -0,0 +1,61 @@
+# compute_tmf Test 15: throws ERROR when ignore_seconds_flag  = T and seconds are present
+
+    Code
+      compute_tmf(dtc = c("2020-11-11T11:11:11", "2020-11-11T11:11"), dtm = ymd_hms(c(
+        "2020-11-11T11:11:11", "2020-11-11T11:11:00")), ignore_seconds_flag = TRUE)
+    Condition
+      Error in `compute_tmf()`:
```

Comment:
So when this happens in the derive_vars_dt() it might confuse the user as they just ran derive_vars_dtm, but comput_tmf comes back - maybe for a different error messaging rework, but at least this function is exported.  
![image](https://github.com/pharmaverse/admiral/assets/10111024/1ecd20db-dde3-400c-8aaf-3a0b2915e0d1)

Reply:
@bms63 , I have added some code to catch the `ignore_seconds_flag` error.